### ISSUE TITLE
Zooming should not be disabled

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3971]
## Area or Component
Seen in the patient experience, but potentially an issue application-wide

## Quick Summary
Zooming and scaling is disabled on the application.

## Accessibility Violation
Zooming must be enabled for low vision users who use screen magnification to interact with the web.
https://dequeuniversity.com/rules/axe/4.4/meta-viewport?application=AxeChrome

## Action Requested
Enable zooming and scaling

## Source Code (optional)
[public/index.html](https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/public/index.html#L17-L17)
